### DISCUSSION TITLE
truetype.h : Inverted glyphs with PackFontRange()

### DISCUSSION THREAD

**Discussion by heysokam** [nothings/stb#1382] [Q&A]
I managed to create a tga font atlas with the use of PackFontRange()
But the font glyphs are being rasterized with Y inverted
I noticed that the call to `stbtt_Rasterize` inside `stbtt_MakeGlyphBitmapSubpixel` uses a hardcoded `1` value for `int invert`
Is there an existing way to change this behavior, or is modifying the library's code required for getting that output?

**Comment by nothings** [comment]
The glyphs are probably not being rasterized with Y inverted. TGA stores images inverted compared to most formats, so you're probably using a TGA writer that doesn't correct for this, or the engine you're reading doesn't handle this correctly. It is not appropriate to change stb_truetype to fix this; either fix the TGA reader/writer, or write your own code to invert the image before writing it.
